packages/fuels/tests/contracts/configurables/src/main.sw

```diff
@@ -12,23 +12,30 @@ struct StructWithGeneric<D> {
 }
 
 configurable {
-    U8: u8 = 8u8,
     BOOL: bool = true,
-    ARRAY: [u32; 3] = [253u32, 254u32, 255u32],
+    U8: u8 = 8,
+    U16: u16 = 16,
+    U32: u32 = 32,
+    U64: u64 = 63,
+    U256: u256 = 0x0000000000000000000000000000000000000000000000000000000000000008u256,
+    B256: b256 = 0x0101010101010101010101010101010101010101010101010101010101010101,
     STR_4: str[4] = __to_str_array("fuel"),
+    TUPLE: (u8, bool) = (8, true),
+    ARRAY: [u32; 3] = [253, 254, 255],
     STRUCT: StructWithGeneric<u8> = StructWithGeneric {
-        field_1: 8u8,
+        field_1: 8,
         field_2: 16,
     },
     ENUM: EnumWithGeneric<bool> = EnumWithGeneric::VariantOne(true),
 }
+//U128: u128 = 128, //TODO: add once https://github.com/FuelLabs/sway/issues/5356 is done
```

Comment:
Is there a task on our side for this?

Reply:
https://github.com/FuelLabs/fuels-rs/issues/1367